assets/css/views/communications.css

```diff
@@ -107,3 +107,86 @@ span.communication__sender__stats,
 .communication--destroyed {
   background-color: var(--destroyed-content-color);
 }
+
+.communication__body__avatar {
+  flex: 0 0 auto;
+  margin-right: 12px;
+}
+
+/* Override padding from block__content, so we can do it on a per-element basis */
+.communication__content {
+  padding: 0;
+}
+
+.communication__body__avatar,
```

Comment:
I think sub-elements are prohibited by BEM, and I was not happy to figure that out. But I see that this rule isn't followed intentionally in this codebase?